### PR TITLE
log instances when destroy server group has not yet completed

### PR DESCRIPTION
**Pull request by anotherchrisberry**
Adding a little visibility into why a destroy task has not yet completed - something we see periodically

@claymccoy please review

### CODE REVIEW

**Comment by anotherchrisberry** [comment]
or @ajordens can review it

File: orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/WaitForDestroyedServerGroupTask.groovy [hunk 1]
```diff
@@ -59,10 +59,12 @@ class WaitForDestroyedServerGroupTask extends AbstractCloudProviderAwareTask imp
...
       }
-      if (!cluster.serverGroups.find { it.name == serverGroupName && it.region == serverGroupRegion }) {
+      def serverGroup = cluster.serverGroups.find { it.name == serverGroupName && it.region == serverGroupRegion }
+      if (!serverGroup) {
         return new DefaultTaskResult(ExecutionStatus.SUCCEEDED)
       }
-
+      def instances = serverGroup.instances?.findResults { it.name }?.join(',') ?: '[none]'
+      log.info("${serverGroupName}: not yet destroyed, found instances: ${instances}")
       return new DefaultTaskResult(ExecutionStatus.RUNNING)
```
Comment (ajordens) [inline orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/WaitForDestroyedServerGroupTask.groovy]: Any reason not to include the remaining instances in the context? ... might be easier to track down than trolling through log messages?

**Comment by ajordens** [comment]
Might want to consider bumping the backoffPeriod up from 1s ... maybe 10s? Not sure we need to be quite that aggressive.

File: orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/WaitForDestroyedServerGroupTask.groovy (hunk 1 above)
Comment (anotherchrisberry) [inline orca-clouddriver/src/main/groovy/com/netflix/spinnaker/orca/clouddriver/tasks/WaitForDestroyedServerGroupTask.groovy]: Good call. I was going for a minimal amount of code with some easy grep-able pattern to trace what was happening. I'll add them to the context, but leave the log messages in place so we can see the history of the instances going away by diving into logs.

**Comment by anotherchrisberry** [comment]
I will bump the backoffPeriod to 10s, also.

**Comment by ajordens** [comment]
LGTM after you `return new DefaultTaskResult(ExecutionStatus.SUCCEEDED, [remainingInstances: []])`

**Comment by ajordens** [comment]
LGTM, feel free to merge once the build passes

**Pull request merged**